rdmo/domain/admin.py

```diff
@@ -11,7 +11,16 @@ class AttributeAdminForm(forms.ModelForm):
 
     class Meta:
         model = Attribute
-        fields = '__all__'
+        fields = [
```

Comment:
This is a fix for the violation against DJ007. What do you think?

Reply:
This is a good idea, I guess.